docs/components/networktransform.md

```diff
@@ -50,28 +45,90 @@ With such nested NetworkTransforms you can theoretically have a NetworkTransform
  _You could have a single root-parent GameObject that has a NetworkObject component and under the root-parent several levels of nested child GameObjects that all have NetworkTransform components attached to them. Each child GameObject would not require a NetworkObject component in order for each respective NetworkTransform component to function/synchronize properly._
 :::
 
+### When to nest NetworkTransforms
+
+If you plan on having unique motion relative to a parent NetworkTransform should you consider nesting NetworkTransforms. As an example, if you are using a NetworkAnimator component to synchronize animations then you need not worry about synchronizing each child node of the model animating as the animation system and NetworkAnimator are handling this for you. 
+
+Really, if you can get away with allowing the parent to synchronize its transform then all child transforms will move, rotate, and scale with the parent automatically. As an example, if you wanted some object to orbit another object and used the server time (i.e. NetworkManager.ServerTime) to feed into something like a sine or cosine based calculation and used that on all instances (local and remote) then you would have no reason to synchronize the motion of the child orbiting the parent since you are feeding the algorithm a value already being synchronized with all clients (i.e. the server time).
+
+:::tip
+**The general rule to follow is:**
+
+Unless you have some unique motion that needs to be applied to a child object and it can't be synchronized through an animation or an algorithm based on the synchronized time, then you should nest NetworkTransforms.
+
+However, nesting NetworkTransforms is a more optimal path then creating a bunch of individual single node network prefabs with NetworkTransforms and parenting those once they are spawned.
```

Comment:
Is this supposed to read 'unless you have [specific requirements], then you _shouldn't_ nest NetworkTransforms'? The paragraph that follows seems to imply that nesting NetworkTransforms should be avoided if it all possible.

Reply:
It is actually "logically" correct...but I see your point about the wording.
The intention here is to provide a general outline of the scenario(s) where you might need to have nested NetworkTransforms.

Maybe this:

> Unless you have some unique motion that needs to be applied to a child object and it can't be synchronized through an animation or an algorithm based on the synchronized time, then you should nest NetworkTransforms.

Should be reworded to this:

When Nesting NetworkTransforms you should first determine if there are any alternative approaches to handle child object motion (_i.e. synchronized through animations or using an algorithm that uses the synchronized network time_) in order to assure you are not consuming unnecessary processing time and bandwidth.

Reply:
Got it, ty 🙏